src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -71,6 +74,12 @@
 /* Replication model to use when creating distributed tables */
 int ReplicationModel = REPLICATION_MODEL_COORDINATOR;
 
+typedef enum
```

Comment:
Maybe moving this to a .h file would be better, as we generally do typedefs in header files.